### DISCUSSION TITLE
Hover on close elements with interactive tooltip. strange behavior

### DISCUSSION THREAD

**Discussion by J4v4Scr1pt** [floating-ui/floating-ui#1939] [Q&A] [answered]
Hello!

I would love some help! 😄

I am trying to create a universal tooltip for my app. 
Have worked great so far but now I'm in a scenario where I cant seem to solve the issue.

I have a tooltip that need to be interactive and should be on several elements close to each other.

The scenario that happens is if you hover from one element to the next you can get stuck, like shown here:
https://gyazo.com/ab6cfd58cc7aa5ed7ab3c730e464105d 

If you put  interactive prop to false, it works as expected.

I have tried to fidlle with the props on safepolygon but nothing seems to solve my issue.
I can use blockPointerEvents but then the tooltip is not interactive anymore and the cirles lose the hover on the tooltip.
I have also tried to fiddle with the buffert but that also makes it to behave a bit wierd.

I have created this sandbox and comment out the code for safepolygon that I have tried with.
https://codesandbox.io/s/currying-mountain-w6zezy?file=/src/Tooltip.jsx

Could some smart person out there please point/help me at the right direction with this😃.

Regards

**Comment by atomiks** [comment, marked as answer]
You just need to set `pointer-events: auto;` on the outer purple div that has the hover state. Or, if disabling `blockPointerEvents`, then remove `pointer-events: none;` from `Wrapper`.

One other problem is that due to the buffer around the polygon, two tooltips can be open simultaneously when moving the cursor slowly horizontally. I think you'd need to use a context that wraps the group and ensure all tooltips close when one is open so only one can be open at any given time.

**Reply by J4v4Scr1pt** [reply]
Ahh smart, damn this is tricky. 😂
Good idea about creating a context wrapper, will look in to that, thanks! 

Another question when am at it..
I have looked on many examples doing this tooltip and would you say it would be better if I created it using a FloatingPortal instead?
What would be the benefits of doing so?
